EightMinuteEmpireLegends/MapLoader/wrongFormatFile.csv

```diff
@@ -0,0 +1 @@
+Ha, hi, hello
```

Comment:
Please leave something like:

This, File, Is, CSV, Instead, Of, JSON
Maploader, Does, Not, Support, This, Formatting, Style

its more maintainable in the future.

Reply:
Sure